src/Dofs/DofHandler.jl

```diff
@@ -12,6 +12,16 @@ Access some grid representation for the dof handler.
 """
 get_grid(dh::AbstractDofHandler)
 
+"""
+    SubDofHandler(dh::AbstractDofHandler, cellset)
```

Comment:
```suggestion
    SubDofHandler(dh::AbstractDofHandler, cellset) <: AbstractDofHandler
```
or not?

Reply:
I suggest not to for now, because e.g. `close!(sdh)` is not supported. 
Since this interface is not fully defined yet, but once this is settled it makes sense to document IMO.